sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExec.scala

```diff
@@ -198,7 +218,19 @@ object NoopMetric extends GpuMetric {
   override def value: Long = 0
 }
 
-final case class WrappedGpuMetric(sqlMetric: SQLMetric) extends GpuMetric {
+final case class WrappedGpuMetric(sqlMetric: SQLMetric, noCompanions: Boolean = false)
```

Comment:
nit: Why the double negatives?  `!noCompainins` works but it is not super clean.  Also companions is kind of a broad term when this is specifically for timing without semaphore. A longer name might be better.

Reply:
refined